src/control/cmd/dmg/pool.go

```diff
@@ -39,8 +39,12 @@ import (
 )
 
 const (
+	// minScmNvmeRatio indicates the minimum storage size ratio SCM:NVMe
+	// (requested on pool creation), warning issued if ratio is lower
```

Comment:
This is an improvement; thank you. What I was hoping for was an explanation for why these values were chosen. If they're arbitrary, that's fine, but if there's some background information that we can provide it would probably be helpful to someone who's looking at these values in the future and wondering why they were chosen or if they can/should be changed.

Reply:
the ratio I think is arbitrary (there was a mention from @NiuYawei that 6% was the minimum recommended for performance but @johannlombardi requested that I put the warning threshold at 1%).
the maximum number of service replicas requested is something I don't have context on, @liw  why is the maximum number of pool service replicas that can be requested set to 13?